Solutions/MicrosoftDefenderForEndpoint/Playbooks/Restrict-MDEUrl/Restrict-MDEUrl-entity-trigger/azuredeploy.json

```diff
@@ -0,0 +1,154 @@
+                },
+                "host": {
+                    "connection": {
+                        "name": "@parameters('$connections')['azuresentinel_1']['connectionId']"
+                    }
+                },
+                "method": "post",
+                "path": "/Incidents/Comment"
+            }
+        },
+        "HTTP": {
+        "runAfter": {},
+        "type": "Http",
+        "inputs": {
+            "authentication": {
+                "audience": "https://api.securitycenter.windows.com/",
+                "type": "ManagedServiceIdentity"
+            },
+            "body": {
+                "action": "AlertAndBlock",
+                "application": "Microsoft Sentinel",
+                "description": "This block command has been made through the Restrict-MDEUrl-entityTrigger. Ran on URL Entity from Sentinel Incident with ARM ID: @{triggerBody()?['IncidentArmID']}. Entity properties: @{triggerBody()?['Entity']?['properties']}",
+                "expirationTime": "@{addDays(utcNow(), 90)}",
+                "indicatorType": "Url",
+                "indicatorValue": "@{triggerBody()?['Entity']?['properties']?['Url']}",
+                "title": "@{guid()}"
+            },
+            "method": "POST",
+            "uri": "https://api.securitycenter.windows.com/api/indicators"
+        }
+    }
+},
+"outputs": {}
+},
+"parameters": {
+"$connections": {
+"value": {
+    "azuresentinel_1": {
+        "connectionId": "[resourceId('Microsoft.Web/connections', variables('MicrosoftSentinelConnectionName'))]",
+        "connectionName": "[variables('MicrosoftSentinelConnectionName')]",
+        "id": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', resourceGroup().location, '/managedApis/Azuresentinel')]",
+        "connectionProperties": {
+            "authentication": {
+                "type": "ManagedServiceIdentity"
+            }
+        }
+    }
+}
+}
+}
+},
+"name": "[parameters('PlaybookName')]",
+"type": "Microsoft.Logic/workflows",
+"location": "[resourceGroup().location]",
+"identity": {
+"type": "SystemAssigned"
+},
+"tags": {
+"hidden-SentinelTemplateName": "Restrict-MDEUrl-entityTrigger",
+"hidden-SentinelTemplateVersion": "1.0"
+},
+"apiVersion": "2017-07-01",
+"dependsOn": [
+"[resourceId('Microsoft.Web/connections', variables('MicrosoftSentinelConnectionName'))]"
+]
+},
+{
+"type": "Microsoft.Web/connections",
+"apiVersion": "2016-06-01",
+"name": "[variables('MicrosoftSentinelConnectionName')]",
+"location": "[resourceGroup().location]",
+"kind": "V1",
+"properties": {
+"displayName": "[variables('MicrosoftSentinelConnectionName')]",
+"customParameterValues": {},
+"parameterValueType": "Alternative",
+"api": {
+"id": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', resourceGroup().location, '/managedApis/Azuresentinel')]"
+}
+}
+}
+]
+}

```

Comment:
1. Can you correct the code indentation?
2. Handle the scenariHandle the scenario when ARMId doesn't exist?